pkgs/by-name/lz/lzfse/package.nix

```diff
@@ -1,29 +1,34 @@
-{ lib, stdenv, fetchFromGitHub, cmake }:
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "lzfse";
   version = "1.0";
 
   src = fetchFromGitHub {
     owner = "lzfse";
     repo = "lzfse";
-    rev = "lzfse-${version}";
-    sha256 = "1mfh6y6vpvxsdwmqmfbkqkwvxc0pz2dqqc72c6fk9sbsrxxaghd5";
+    rev = "lzfse-${finalAttrs.version}";
+    hash = "sha256-pcGnes966TSdYeIwjJv4F7C++cRzuYorb7rvu4030NU=";
   };
 
   nativeBuildInputs = [ cmake ];
 
-  meta = with lib; {
+  meta = {
     homepage = "https://github.com/lzfse/lzfse";
     description = "Reference C implementation of the LZFSE compressor";
     longDescription = ''
       This is a reference C implementation of the LZFSE compressor introduced in the Compression library with OS X 10.11 and iOS 9.
       LZFSE is a Lempel-Ziv style data compression algorithm using Finite State Entropy coding.
       It targets similar compression rates at higher compression and decompression speed compared to deflate using zlib.
     '';
-    platforms = platforms.unix;
-    license = licenses.bsd3;
-    maintainers = [ ];
+    platforms = lib.platforms.unix;
+    license = lib.licenses.bsd3;
+    maintainers = with lib.maintainers; [ KSJ2000 ];
```

Comment:
You're missing from `maintainers/maintainer-list.nix`. Please add yourself ([instructions](https://github.com/NixOS/nixpkgs/tree/master/maintainers#how-to-become-a-maintainer)), taking care to use a separate commit. I encourage you to use this same PR to add yourself to the maintainers list.

Reply:
@KSJ2000 please see above :)

Reply:
Added in https://github.com/NixOS/nixpkgs/pull/348979, waiting on [a rebase](https://github.com/NixOS/nixpkgs/pull/348979#issuecomment-2475643855)